src/Bicep.Core/Semantics/Namespaces/AzNamespaceType.cs

```diff
@@ -18,6 +18,13 @@ public static class AzNamespaceType
     {
         public const string BuiltInName = "az";
 
+        public static NamespaceSettings Settings { get; } = new(
+            IsSingleton: true,
+            BicepProviderName: BuiltInName,
+            ConfigurationType: null,
+            ArmTemplateProviderName: "AzureResourceManager",
```

Comment:
>AzureResourceManager [](http://example.com/codeflow?start=38&length=20)

How are these values used? (Haven't read the changes to the end yet.)

Reply:
They're used to generate the "import" statements when emitting.